pwm/user_data/user_data.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -x
+exec > >(tee /var/log/user-data.log|logger -t user-data ) 2>&1
+echo BEGIN
+date '+%Y-%m-%d %H:%M:%S'
+
+# Setup ECS
+echo "ECS_CLUSTER=${ecs_cluster_name}" >> /etc/ecs/ecs.config
+service docker start
+start ecs
+
+# Install tools
+sudo -i
+yum install -y git wget yum-utils jq
+easy_install pip
+PATH=/usr/local/bin:$PATH
+pip install ansible==2.6 virtualenv awscli boto botocore boto3 passlib
+
+# Download user inventory and run ansible
+/usr/bin/curl -o ~/users.yml https://raw.githubusercontent.com/ministryofjustice/hmpps-delius-ansible/master/group_vars/dev.yml
+sed -i '/users_deleted:/,$d' ~/users.yml
```

Comment:
Should this be removed? Do we want to leave the default users (ec2-user or centos) on?

Reply:
According to Brett, the users_deleted part of the users role breaks on the ECS AMI - so recommendation was to remove it from the users.yml file

Reply:
a that would make sense. A comment to explain this would be useful for those coming along later :)